cellprofiler/modules/saveimages.py

```diff
@@ -745,6 +759,9 @@ def save_image(self, workspace):
             if image.volumetric and self.file_format.value == FF_TIFF:
                 save_kwargs.update({"compress": 6})
 
+            if self.tiff_compress.value and self.file_format.value == FF_TIFF:
```

Comment:
Looks like the compression is still forced on volumetric images by the block above? If this is the case we should either hide the setting when in 3D mode or make compression optional on volumetric as well.

Reply:
I'm not sure how to hide a setting only in volumetric mode (or change its default only in volumetric mode)- to know if a pipeline is working on 3D images or not, you need the pipeline or the workspace, neither of which is passed into the settings by default.  Do you know a way to work around this? Otherwise, it's a bit messy, but we could say "this setting will be ignored on 3D images, for which compression is always used".

Reply:
Looks like you're right, the modules themselves only have access to workspace info in certain functions. With that in mind I'd say that leaving a note in the help text is fine. Alternatively we could default to 'on' when upgrading and make the setting dictate what to do even in volumetric.

We should probably merge the two `if` blocks anyway to avoid problems down the line, just in case we ever want to change compression method.